openedx/features/custom_fields/multiselect_with_other/forms/fields.py

```diff
@@ -1,5 +1,5 @@
 """This module contains MultiSelect form field with other option"""
-
+# pylint: disable=E1101
```

Comment:
1. Please disables error on the relevant line (i.e. **line # 30**), 
2. Always prefer error-name over error-code i.e.e **no-member** over **E1101**

```
self.error_messages.update(
    dict(invalid_length=_(  # pylint: disable=no-member
        'Other field value, maximum allowed length violation. Allowed limit is upto {other_max_length}'
        ' characters.').format(
        other_max_length=other_max_length)))
```

Reply:
Corrected it.